e2etest/provider_cfg/all/Dockerfile

```diff
@@ -1,17 +1,16 @@
 # Copyright 2021 Contributors to the Parsec project.
 # SPDX-License-Identifier: Apache-2.0
 
-FROM  ghcr.io/parallaxsecond/parsec-service-test-all 
+FROM   parsec-ci-service-test-all
 
 # Install Rust toolchain for root
 USER root
 
-# Download the SPIRE server and agent
-RUN curl -s -N -L https://github.com/spiffe/spire/releases/download/v0.11.1/spire-0.11.1-linux-x86_64-glibc.tar.gz | tar xz
```

Comment:
Are we not using this anymore?

Reply:
it is now in the downloaded dockerfile does as the parsec docker build

Reply:
part of the docker image creation downloaded from parsec server repo